meteor-app/imports/ui/actions/index.js

```diff
@@ -8,4 +8,4 @@ export * from './workspace_change_layer_opacity';
 export * from './workspace_inspect_point';
 export * from './charts_set_filters';
 export * from './charts_inspect_point_resolve_data';
-export * from './workspace_close_welcome_window';
+export * from './workspace_switch_welcome_window';
```

Comment:
P.S. "Switch" is when you "switch windows", "switch tabs", where there is no addition or removal of entities.